core/trino-main/src/main/java/io/trino/eventlistener/DefaultEventListenerManager.java

```diff
@@ -0,0 +1,178 @@
+/*
```

Comment:
> Extract as interface to make it easier to later introduce a chain
of responsibility.

Somehow english here is odd. Can you please improve wording?